pkg/aspect/quotasManager.go

```diff
@@ -194,7 +170,10 @@ func quotaDefinitionFromProto(desc *dpb.QuotaDescriptor) (*adapter.QuotaDefiniti
 		labels[label.Name] = l
 	}
 
-	dur, _ := ptypes.DurationFromProto(desc.Expiration)
+	dur, err := ptypes.DurationFromProto(desc.Expiration)
```

Comment:
We'll also need to update //pkg/config/proto and use and generate a gogo version rather than a regular old go version. Today we parse our config into a proto struct generated with protoc due to golang's broken handling of `Struct` protos; AFAIK no reason we couldn't use gogo's struct instead. @mandarjog